samples/excel/pivot-tables/refresh-pivot-table.yaml

```diff
@@ -0,0 +1,166 @@
+id: ''
+gist: ''
+source: EXCEL
+author: Microsoft
+name: Refresh pivot table
+description: |
+  Refresh pivot table
+script:
+  content: |+
+    $('#create-table').click(createTable);
+    $('#refresh-pivot-table').click(refreshPivotTable);
+
+    /** Create a new table with some sample data */
+    function createTable() {
+        Excel.run(async (context) => {
+            let sheet: Excel.Worksheet;
+            try {
+                //Call a helper to delete the playground sheet if it exists
+                deleteIfSheetExists("Playground");
+
+                // Queue a command to add a new worksheet
+                sheet = context.workbook.worksheets.add("Playground");
+
+                // Call the helper to create the table
+                await queueCreateTable(sheet);
+
+                //Queue a command to activate the sheet
+                sheet.activate();
+
+                // Run all of the queued up commands
+                await context.sync();
+            }
+            catch (error) {
+                 OfficeHelpers.Utilities.log(error);
+            }
+        })
+    }
+
+
+    /** Refresh pivot table in the sheet */
+    function refreshPivotTable() {
+        Excel.run(async (context) => {
+            let sheet: Excel.Worksheet;
+            try {
+                // Queue a command to get the sheet
+                sheet = context.workbook.worksheets.getActiveWorksheet();
+
+                // Call the helper to sort the table
+                await queueRefreshPivotTable(sheet);
+                
+                // Run all of the queued up commands
+                await context.sync();
+            }
+            catch (error) {
+                 OfficeHelpers.Utilities.log(error);
+            }
+        })
+    }
+
+    /** Create the table */ 
+    async function queueCreateTable(sheet: Excel.Worksheet) {
+
+            // Queue a command to add a table to the sheet
+            let expensesTable = sheet.tables.add('A1:D1', true);
+            expensesTable.name = "ExpensesTable";
+
+            // Queue a command to set the header row
+            expensesTable.getHeaderRowRange().values = [["DATE", "MERCHANT", "CATEGORY", "AMOUNT"]];
+
+            // Queue a command to add rows to the table
+            expensesTable.rows.add(null, [
+                ["1/1/2017", "THE PHONE COMPANY", "Communications", "$120"],
```

Comment:
Will "1/2/2017" work on a German machine?  It will almost certainly reverse month and day.

I think the right pattern is:
* Set number format to whatever you desire (though make it Europe-friendly)
* Set the values as an ISO-esque string "2017-02-01"

I'm not sure this works.  But sure seems like a good thing to document one way or another.